RedacteurPortaal.Api/DTOs/NewsItemDTO.cs

```diff
@@ -1,30 +1,47 @@
-﻿using System;
+﻿using RedacteurPortaal.Api.DTOs;
+using RedacteurPortaal.DomainModels.Media;
+using RedacteurPortaal.DomainModels.NewsItem;
+using RedacteurPortaal.DomainModels.Shared;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace RedacteurPortaal.Api.Models;
 
+[Serializable]
 public class NewsItemDto
 {
-    public Guid? Id { get; set; }
-
-    public string ApprovalStatus { get; set; }
+    public Guid Id { get; set; }
 
     public string? Title { get; set; }
 
-    public string? Status { get; set; }
+    public Status? Status { get; set; }
```

Comment:
Als je dit de enum maakt ipv string, krijg je een nummer terug. Daarom had ik het als string gezet, lijkt mee voor de frontend een stuk makkelijker als het "Pending" is ipv 1